distribution/std-lib/Base/src/Net/Http.enso

```diff
@@ -0,0 +1,250 @@
+from Base import all
+import Base.Net.Proxy
+import Base.Net.Uri
+import Base.Net.Http.Auth
+import Base.Net.Http.Form
+import Base.Net.Http.Header
+import Base.Net.Http.Request
+import Base.Net.Http.Request.Body as Request_Body
+import Base.Net.Http.Response
+import Base.System.File
+import Base.Time.Duration
+import Base.Time.Time
+
+polyglot java import java.time.Duration as Java_Duration
+polyglot java import java.net.InetSocketAddress
+polyglot java import java.net.ProxySelector
+polyglot java import java.net.URI
+polyglot java import java.net.http.HttpClient
+polyglot java import java.net.http.HttpRequest
+polyglot java import java.net.http.HttpResponse
+polyglot java import org.enso.base.Http_Utils
+
+type Http
+
+    type Http timeout auth follow_redirects proxy
+
+    ## Send an OPTIONS request.
+
+       > Example
+         Send an OPTIONS request.
+             Http.new.options "http://httpbin.org"
```

Comment:
Is there a particular reason that examples point to `http` instead of `https`?

Reply:
No. I didn't think about that